lametro/templates/lametro/legislation.html

```diff
@@ -24,6 +24,7 @@ <h1>
                     {% if legislation.inferred_status %}
                         {{ legislation.inferred_status | inferred_status_label | safe }}
                     {% endif %}
+                <a href="rss/" title="RSS feed for Legislation {{legislation.friendly_name}}" target="_blank"><i class="fa fa-rss-square" aria-hidden="true"></i></a>
```

Comment:
Oops, this got removed in https://github.com/datamade/la-metro-councilmatic/pull/645 and I must have missed the change when I prepped this branch for you initially. You can remove this! 

```suggestion
```